xernel/kernel/src/sched/process.rs

```diff
@@ -34,7 +33,6 @@ pub struct Process {
     pub threads: Vec<Arc<Spinlock<Thread>>>,
     pub fds: BTreeMap<usize, File>,
     pub kernel_thread_stack_top: usize,
-    pub user_thread_stack_top: usize,
```

Comment:
Elaborate removal

Reply:
User thread stack is now stored as a mapping in the Vm at the highest available address